### PR TITLE
Redisinsight v1.14.0 release notes

### PR DESCRIPTION
**Pull request by GnaneshKunal**
NOTE: Don't merge until RedisInsight v1.13.2 release.

### CODE REVIEW

File: content/ri/release-notes/v1.13.0.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
 weight: 7
 ---
 
+## 1.13.2 (May 2023)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.2)!
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:11]:
```suggestion
This is the maintenance release of RedisInsight 1.13 (v1.13.2).
```

File: content/ri/release-notes/v1.13.0.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
...
+
+### Fixes:
+- Core:
+  - Added support for exporting connections to be used in RedisInsight V2.
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:15]:
```suggestion
  - Added support for exporting connections to RedisInsight v2.
```

File: content/ri/release-notes/v1.13.0.md [hunk 3]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
...
+  - Fixed Promt verification bug for sentinel instances.
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:16]:
```suggestion
  - Fixed Prompt verification bug for Sentinel instances.
```

File: content/ri/release-notes/v1.13.0.md [hunk 4]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
...
+- Memory Analysis:
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:17]:
```suggestion
- Memory analysis:
```

File: content/ri/release-notes/v1.13.0.md [hunk 5]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
...
+  - Add support for `setlistpack` and `streamlistpack3` Redis 7 encoding types parsing.
```
Comment (nermiller) [inline content/ri/release-notes/v1.13.0.md:18]:
```suggestion
  - Added support for `setlistpack` and `streamlistpack3` Redis 7 encoding types parsing.
```

**Review by nermiller — COMMENTED** [review]
A bit of feedback for you @GnaneshKunal

File: content/ri/release-notes/v1.13.0.md (hunk 2 above)
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v1.13.0.md:15]: Added support for exporting database connections to easily migrate them to RedisInsight v2 by bulk exporting to a file.

File: content/ri/release-notes/v1.13.0.md [hunk 6]
```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
 weight: 7
 ---
 
+## 1.13.2 (May 2023)
```
Comment (ViktarStarastsenka) [inline content/ri/release-notes/v1.13.0.md:9]: 1.14.0? because of the export database connections

File: content/ri/release-notes/v1.13.0.md (hunk 1 above)
Comment (ViktarStarastsenka) [reply content/ri/release-notes/v1.13.0.md:11]: RedisInsight version 1.X has been retired on April 30, 2023, and will no longer be supported.
To continue using the best RedisInsight features and capabilities, download the latest RedisInsight version 2.Y from our [website](https://redis.com/redis-enterprise/redis-insight/) or install it from any app store.

This is the maintenance release of RedisInsight 1.14 (v1.14.0).

File: content/ri/release-notes/v1.14.0.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+## Headlines
+- Export connections to RedisInsight v2.
+
+## Details
+
+### Core
+  - Added support for exporting database connections to easily migrate them to RedisInsight v2 by bulk exporting to a file.
+  - Fixed Prompt verification bug for Sentinel instances.
+### Memory analysis
```
Comment (nermiller) [inline content/ri/release-notes/v1.14.0.md:24]:
```suggestion

### Memory analysis
```

**Review by nermiller — APPROVED** [review]

**Comment by GnaneshKunal** [comment]
I'll post here when Redisinsight v1.14.0 is deployed. Please merge post that.
________________________________

**Comment by nermiller** [comment]
@GnaneshKunal yup, just let us know

**Comment by GnaneshKunal** [comment]
RedisInsight v1.14.0 has been released.

**Pull request merged**